src/AccordionItemTitle/AccordionItemTitle.tsx

```diff
@@ -44,43 +44,24 @@ export default class AccordionItemTitle extends React.Component<
 
         const id = `accordion__title-${uuid}`;
         const ariaControls = `accordion__body-${uuid}`;
-        const role = accordion ? 'tab' : 'button';
```

Comment:
Now 'accordion' is an unused prop, so it needs to be removed from this 'destructuring assignment' and also from this component's type definition.

Reply:
This was already simplified to `const role = 'button'`, unsure why you're seeing it this way here 🤔

Reply:
I don't follow.. I'm just pointing out that now that the `role` is a fixed string literal, then the AccordionItemTitle is not dependent on the `accordion: AccordionContainer` prop. So it need to be removed from lines 10 and 37 (and line 2 and 72 in `AccordionItemTitle.wrapper` etc etc).